pkg/operator/staticpod/startupmonitor/assets/startup-monitor-pod.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: {{.TargetNamespace}}
+  name: {{.TargetName}}-startup-monitor
+  labels:
+    revision: "REVISION"
+spec:
+  containers:
+    - name: startup-monitor
+      image: {{.OperatorImage}}
+      imagePullPolicy: IfNotPresent
+      terminationMessagePolicy: FallbackToLogsOnError
+      command: ["cluster-kube-apiserver-operator", "startup-monitor"] # todo: make it dynamic
+      args:
+        - {{.Verbosity}}
+        - --fallback-timeout-duration=300s
+        - --target-name=kube-apiserver
+        - --manifests-dir=/etc/kubernetes/manifests
+        - --resource-dir=/etc/kubernetes/static-pod-resources
+        - --revision=REVISION
```

Comment:
we really still have `REVISION` and not `{{.Revision}}` ?

Reply:
yeah, the `REVISION` is replaced by the `installer-pod`

Reply:
at some point we should clean up :)